### PR TITLE
[PRM] Changes made for demo that should be on master

### PR DESCRIPTION
**Pull request by jairmedeiros**

### CODE REVIEW

**Comment by ytuik** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/02_home/01_announcements/page-definition.json [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+										"emailAssetEntryAddedBody_ca_ES": "Dear [$TO_NAME$],<br /><br />\n\nThis is an autogenerated email for the [$PORTLET_NAME$] portlet.<br /><br />\n\nThe following asset entries have been added to [$PORTLET_TITLE$]: [$ASSET_ENTRIES$].<br /><br />\n\nSincerely,<br />\n[$FROM_NAME$]<br />\n[$FROM_ADDRESS$]",
+										"emailAssetEntryAddedBody_de_DE": "Dear [$TO_NAME$],<br /><br />\n\nThis is an autogenerated email for the [$PORTLET_NAME$] portlet.<br /><br />\n\nThe following asset entries have been added to [$PORTLET_TITLE$]: [$ASSET_ENTRIES$].<br /><br />\n\nSincerely,<br />\n[$FROM_NAME$]<br />\n[$FROM_ADDRESS$]",
+										"emailAssetEntryAddedBody_en_US": "<p>Dear [$TO_NAME$],<br />\n<br />\nThis is an autogenerated email for the [$PORTLET_NAME$] portlet.<br />\n<br />\nThe following asset entries have been added to [$PORTLET_TITLE$]: [$ASSET_ENTRIES$].<br />\n<br />\nSincerely,<br />\n[$FROM_NAME$]<br />\n[$FROM_ADDRESS$]</p>\n",
```
Comment (ytuik) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/02_home/01_announcements/page-definition.json:155]: I don't think that this <p> tag is necessary

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/detail-view-table/index.html [hunk 2]
```diff
@@ -11,15 +11,15 @@
 	</thead>
 
 	<tbody class="ml-1">
-		[#list 0..configuration.numberOfRows-1 as i]
+		[#list 0..configuration.numberOfRows as index]
```
Comment (ytuik) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/detail-view-table/index.html:14]: [#list 1..configuration.numberOfRows as index]
as we are removing the -1 the count of the list has to start at 1

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/mdf-request-details-summary/index.js [hunk 3]
```diff
@@ -64,4 +59,12 @@ const formatCurrency = (value) =>
 		style: 'currency',
 	}).format(value);
 
-updateMDFDetailsSummary();
+const formatDate = (value) =>
```
Comment (ytuik) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/mdf-request-details-summary/index.js:62]: It's missing the year formating

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/announcements-subscribe/fragment.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+	"configurationPath": "index.json",
```
Comment (ytuik) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/announcements-subscribe/fragment.json:2]: Change the configuration archive name to configuration.json to keep the pattern

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-claim-details-summary/index.js [hunk 5]
```diff
@@ -56,4 +58,4 @@ const formatCurrency = (value) =>
 		style: 'currency',
 	}).format(value);
 
-updateMDFClaimSummary();
+getMDFClaimSummary();
```
Comment (ytuik) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-claim-details-summary/index.js:61]: if (layoutMode !== 'edit') {
     getMDFClaimSummary();
}

add this verification to follow the pattern

**Comment by jackelyneaquino** [comment]
jackelyneaquino

**Review by ytuik — COMMENTED** [review]
request changes

**Pull request closed without merging**